includes/abstracts/abstract-wc-order.php

```diff
@@ -739,11 +739,24 @@ public function get_shipping_methods() {
 	}
 
 	/**
+	 * Sets shipping method title.
+	 *
+	 * @param array $shipping_method
+	 */
+	public function set_shipping_method( $shipping_method ) {
+		$this->set_prop( 'shipping_method', $shipping_method );
+	}
+
+	/**
 	 * Gets formatted shipping method title.
 	 *
 	 * @return string
 	 */
-	public function get_shipping_method() {
+	public function get_shipping_method( $context = 'view' ) {
+		if ( 'edit' === $context ) {
```

Comment:
See [this explanation](https://github.com/woocommerce/woocommerce/pull/16713/commits/ff9316b093db7e9890555e7e802a57db204e5c31#diff-9b4164165828b26c4b7aec01c7b17884R763) for the reason of this change.